src/app/Fake.IO.FileSystem/Shell.fs

```diff
@@ -337,6 +339,14 @@ module Shell =
             f.MoveTo(targetName) |> ignore
         | FileSystemInfo.Directory _ -> () //TODO: logVerbosefn "Ignoring %s, because it is a directory." fileName
 
+    let private moveDir target fileName =
+        let fi = FileSystemInfo.ofPath fileName
+        match fi with
+        | FileSystemInfo.Directory (d, _) ->
+            let targetName = target @@ fi.Name
+            d.MoveTo(targetName) |> ignore
```

Comment:
can we write `ignore<DirectoryInfo>` here (or whatever `MoveTo` returns here)?

Reply:
The method returns `void`, so I guess we don't need to `ignore`?

Reply:
Indeed